contracts/sdk/bubblegum/src/generated/types/LeafSchema.ts

```diff
@@ -61,7 +61,7 @@ export const leafSchemaBeet = beet.dataEnum<LeafSchemaRecord>([
         ['dataHash', beet.uniformFixedSizeArray(beet.u8, 32)],
         ['creatorHash', beet.uniformFixedSizeArray(beet.u8, 32)],
       ],
-      'LeafSchemaRecord["V0"]'
+      'LeafSchemaRecord["V1"]'
     ),
   ],
-]) as beet.FixedSizeBeet<LeafSchema> | any
+]) as beet.FixedSizeBeet<LeafSchema>
```

Comment:
This is a solita bug (I think). Without the `| any` hack I get the following error:
```
FixedSizeBeet<EnumDataVariant<"V1", { id: PublicKey; owner: PublicKey; delegate: PublicKey; nonce: bignum; dataHash: number[]; creatorHash: number[]; }>, Partial<...>>; toFixedFromValue(val: any): FixedSizeBeet<...>; description: string; }' to type 'FixedSizeBeet<{ __kind: "V1"; } & { id: PublicKey; owner: PublicKey; delegate: PublicKey; nonce: bignum; dataHash: number[]; creatorHash: number[]; }, Partial<{ __kind: "V1"; } & { ...; }>>' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Type '{ toFixedFromData(buf: Buffer, offset: number): FixedSizeBeet<EnumDataVariant<"V1", { id: PublicKey; owner: PublicKey; delegate: PublicKey; nonce: bignum; dataHash: number[]; creatorHash: number[]; }>, Partial<...>>; toFixedFromValue(val: any): FixedSizeBeet<...>; description: string; }' is not comparable to type 'ElementCollectionFixedSizeBeet<{ __kind: "V1"; } & { id: PublicKey; owner: PublicKey; delegate: PublicKey; nonce: bignum; dataHash: number[]; creatorHash: number[]; }, Partial<{ __kind: "V1"; } & { ...; }>>'.
  ```
  Didn't want to think about it, but it's probably worth resolving at some point